imguiWrapper.h

```diff
@@ -7,61 +7,211 @@ extern "C"
 {
 #endif
 
+/*******************************/
+/* Context creation and access */
+/*******************************/
 extern IggContext iggCreateContext(IggFontAtlas sharedFontAtlas);
 extern void iggDestroyContext(IggContext context);
 extern IggContext iggGetCurrentContext();
 extern void iggSetCurrentContext(IggContext context);
+//DebugCheckVersionAndDataLayout
 
-extern IggIO iggGetCurrentIO(void);
-extern IggGuiStyle iggGetCurrentStyle(void);
+/*************************/
+/* Main - v1.67 complete */
```

Comment:
I'd avoid adding concrete version numbers, as they have a high potential of becoming invalid over time.